crates/biome_html_parser/tests/html_specs/ok/cdata.html

```diff
@@ -0,0 +1 @@
+<![CDATA[example]]>
```

Comment:
Can we add some more complex examples, like: `<![CDATA[<div attributes="some-attribute"></div>]]>`

Reply:
Sure. It'll also be better covered when we bring in prettier's tests.